script/CubitExport-release.cmake

```diff
@@ -0,0 +1,190 @@
+#----------------------------------------------------------------
+# Generated CMake target import file for configuration "Release".
+#----------------------------------------------------------------
```

Comment:
I think I asked this somewhere else, and I don't remember if it was answered.... is this file supposed to be in the SDK?  Wasn't it in the SDK in the past?

Reply:
in this case nop, but in some yes. (small fix need to be done for macOS especially...)